src/Input.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Input({ newTodo, onChange }) {
+  return (
+    <input type="text" placeholder="할일을 입력해 주세요" value={newTodo || ''} onChange={onChange} />
```

Comment:
```suggestion
    <input
      type="text"
      placeholder="할일을 입력해 주세요"
      value={newTodo || ''}
      onChange={onChange}
    />
```

한 줄이 길어질 땐, 여러 줄로 표현하면 더 보기 편할 것 같습니다~